src/pages/Home/components/Hero.styles.js

```diff
@@ -0,0 +1,43 @@
+import styled, { css } from 'styled-components';
```

Comment:
The button and heatmap are slightly off. I think decreasing the bottom margin of the SubTitle and the to margin of the heatmap each by 1px will solve this.

![image](https://user-images.githubusercontent.com/9722792/78039175-12f71180-736e-11ea-8727-f6343306469e.png)